assets/css/custom/custom-layout-buttons.scss

```diff
@@ -10,6 +14,11 @@
 
 /* Buttons - used across multiple pages */
 
+$dark-orange-colour: #d35400;
```

Comment:
👍 Great point, as these elements become more and more visible I'll be extracting them into separate file(s), importing them and creating new PRs - thanks @PaNaVTEC